### ISSUE TITLE
sbctl not displaying pod logs

### ISSUE THREAD

**Issue by bagel-dawg** [replicatedhq/sbctl#57]
Support bundles from our app cannot display the pod's logs. Is this something that can be enabled? Are we also able to capture the logs of system pods as well?

```
ryan.knauer@RyanKnauer-Lenovo:~$ kubectl logs openebs-localpv-provisioner-5945d7879f-g44np -n openebs
2022/10/17 21:21:52 http: superfluous response.WriteHeader call from github.com/replicatedhq/sbctl/pkg/api.(*logRecord).WriteHeader (middleware.go:20)
                                                                                                                                                      log files not found in support-bundle.
/tmp/sbctl-1447890770/support-bundle/cluster-resources/pods/logs/openebs/openebs-localpv-provisioner-5945d7879f-g44np/openebs-provisioner-hostpath.log
/tmp/sbctl-1447890770/support-bundle/cluster-resources/pods/logs/openebs/openebs-localpv-provisioner-5945d7879f-g44np/openebs-provisioner-hostpath-logs-errors.log
```

**Comment by xavpaice** [comment]
Hi,

Pod logs from the 'logs' collector are stored in a different location - we're working on a fix for this in https://github.com/replicatedhq/troubleshoot/issues/744.

**Comment by xavpaice** [comment]
This should be resolved now with 0.49.1, the logs collector stores logs in the same place as cluster-resources plus a symlink to the 'old' location for backward compatibility.
